README.md

```diff
@@ -71,11 +71,39 @@ jobs:
           command: myscript
 ```
 
-### Run MATLAB Build Across Different Platforms
+### Use MATLAB Batch Licensing Token
+On a GitHub-hosted runner, you need a [MATLAB batch licensing token](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/alternates/non-interactive/MATLAB-BATCH.md#matlab-batch-licensing-token) if your project is private or if your workflow includes transformation products, such as MATLAB Coder&trade; and MATLAB Compiler&trade;. Batch licensing tokens are strings that enable MATLAB to start in noninteractive environments. You can request a token by submitting the [MATLAB Batch Licensing Pilot](https://www.mathworks.com/support/batch-tokens.html) form. 
+
+To use a MATLAB batch licensing token:
+
+1. Set the token as a secret. For more information about secrets, see [Using secrets in GitHub Actions](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions).
+2. Map the secret to an environment variable named `MLM_LICENSE_TOKEN` in your workflow. 
+
+For example, use the latest release of MATLAB on a GitHub-hosted runner to run the tests in your private project. To install the latest release of MATLAB on the runner, specify the **Setup MATLAB** action in your workflow. To run the tests, specify the [Run MATLAB Tests](https://github.com/matlab-actions/run-tests/) action. In this example, `MyToken` is the name of the secret that holds the batch licensing token.
+
+```YAML
+name: Use MATLAB Batch Licensing Token
+on: [push]
+env:
+  MLM_LICENSE_TOKEN: ${{ secrets.MyToken }}
+jobs:
+  my-job:
+    name: Run MATLAB Tests in Private Project
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+      - uses: actions/checkout@v4
+      - name: Set up MATLAB
+      - uses: matlab-actions/setup-matlab@v2
+      - name: Run tests
+      - uses: matlab-actions/run-tests@v2
+```
```

Comment:
Need to remove the dashes before `uses`:
```YAML
name: Use MATLAB Batch Licensing Token
on: [push]
env:
  MLM_LICENSE_TOKEN: ${{ secrets.MyToken }}
jobs:
  my-job:
    name: Run MATLAB Tests in Private Project
    runs-on: ubuntu-latest
    steps:
      - name: Check out repository
        uses: actions/checkout@v4
      - name: Set up MATLAB
        uses: matlab-actions/setup-matlab@v2
      - name: Run tests
        uses: matlab-actions/run-tests@v2
```

Reply:
Done.